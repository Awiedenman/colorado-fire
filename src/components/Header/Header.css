* {
  margin: 0;
}

.header {
  display: flex;
  background-image: url('../../images/trees.jpg');
  background-repeat: repeat;
  background-position: top;
  background-size: 300px 190px;
  height: 152px;
  color: #000;
}

.layer {
  display: flex;
  justify-content: space-around;
  background-color: rgba(230, 221, 221, 0.863);
  border-bottom: 0.5rem double #052709;
  height: 144px;
  width: 100vw;
}

.main-title {
  font-size: 5rem;
  color: #435245;
  padding-top: 1rem;
  font-family: 'Open Sans', sans-serif;
}

.header-navLinks {
  padding: 82px;
}

a {
  text-decoration: none;
  color: #052709;
  font-size: 1.5rem;
  margin: 15px;
}

a:hover {
  color: rgba(255, 72, 0, 0.63);
  border-bottom: .1rem solid rgba(107, 69, 18, 0.767);
  font-weight: bolder;
  margin: 14.85px;
}

.fire {
  color: rgba(255, 72, 0, 0.603);
}

@media screen and (max-width: 1100px) {
  .layer{
    display: flex;
    align-items: center;
    flex-direction: column-reverse;
  }

  .main-title {
    position: inline;
    top: 1.7rem; 
    color: #435245; 
    padding-bottom: 2rem;
    font-family: 'Open Sans', sans-serif;
  }

  .main-title {
    padding: 0;
  }

  .header-navLinks {
    padding: 0 0 10px 0;
  }
}