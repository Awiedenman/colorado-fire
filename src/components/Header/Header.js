import React from 'react';
import { NavLink } from 'react-router-dom';
import './Header.css';


export const Header = () => {
  return ( 
    <div className='header'>
      <h1 className='main-title'> WildFire USA </h1>
      <div className='header-navLinks'>
        <NavLink exact to="/">Current Fires</NavLink>
        <NavLink exact to = "/reportFires" > Report a Fire </NavLink>
      </div>
    </div>
  )
}
export default Header;