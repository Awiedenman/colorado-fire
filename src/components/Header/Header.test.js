import React from 'react'
import { Header } from './Header';
import { shallow } from 'enzyme'

describe('Header', () => {
  test('should match the snapshot', () => {
    const wrapper = shallow(<Header />)

    expect(wrapper).toMatchSnapshot();
  })
})