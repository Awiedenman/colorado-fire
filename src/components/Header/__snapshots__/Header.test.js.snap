// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1
          className="main-title"
>
           WildFire USA 
</h1>,
        <div
          className="header-navLinks"
>
          <NavLink
                    activeClassName="active"
                    aria-current="page"
                    exact={true}
                    to="/"
          >
                    Current Fires
          </NavLink>
          <NavLink
                    activeClassName="active"
                    aria-current="page"
                    exact={true}
                    to="/reportFires"
          >
                     Report a Fire 
          </NavLink>
</div>,
      ],
      "className": "header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " WildFire USA ",
          "className": "main-title",
        },
        "ref": null,
        "rendered": " WildFire USA ",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <NavLink
              activeClassName="active"
              aria-current="page"
              exact={true}
              to="/"
>
              Current Fires
</NavLink>,
            <NavLink
              activeClassName="active"
              aria-current="page"
              exact={true}
              to="/reportFires"
>
               Report a Fire 
</NavLink>,
          ],
          "className": "header-navLinks",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": "Current Fires",
              "exact": true,
              "to": "/",
            },
            "ref": null,
            "rendered": "Current Fires",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": " Report a Fire ",
              "exact": true,
              "to": "/reportFires",
            },
            "ref": null,
            "rendered": " Report a Fire ",
            "type": [Function],
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="main-title"
>
             WildFire USA 
</h1>,
          <div
            className="header-navLinks"
>
            <NavLink
                        activeClassName="active"
                        aria-current="page"
                        exact={true}
                        to="/"
            >
                        Current Fires
            </NavLink>
            <NavLink
                        activeClassName="active"
                        aria-current="page"
                        exact={true}
                        to="/reportFires"
            >
                         Report a Fire 
            </NavLink>
</div>,
        ],
        "className": "header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " WildFire USA ",
            "className": "main-title",
          },
          "ref": null,
          "rendered": " WildFire USA ",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <NavLink
                activeClassName="active"
                aria-current="page"
                exact={true}
                to="/"
>
                Current Fires
</NavLink>,
              <NavLink
                activeClassName="active"
                aria-current="page"
                exact={true}
                to="/reportFires"
>
                 Report a Fire 
</NavLink>,
            ],
            "className": "header-navLinks",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": "Current Fires",
                "exact": true,
                "to": "/",
              },
              "ref": null,
              "rendered": "Current Fires",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": " Report a Fire ",
                "exact": true,
                "to": "/reportFires",
              },
              "ref": null,
              "rendered": " Report a Fire ",
              "type": [Function],
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
